client/src/components/ElementEditor/ElementEditor.js

```diff
@@ -128,50 +118,13 @@ ElementEditor.propTypes = {
   }),
 };
 
-const defaultElementFormState = {};
-
-// Use a memoization to prevent mapStateToProps() re-rendering on formstate changes
-// Any formstate change, including unrelated ones such as from another FormBuilderLoader component
-// will trigger the ElementalEditor to re-render
-const elementFormSelector = createSelector([
-  (state) => {
-    const elementFormState = state.form.formState.element;
-
-    if (!elementFormState) {
-      // This needs to a reference to the defaultElementFormState variable rather than a new object
-      // or redux will think the state has changed and cause the component to re-render
-      return defaultElementFormState;
-    }
-
-    return elementFormState;
-  }], (elementFormState) => {
-  const formNamePattern = loadElementFormStateName('[0-9]+');
-
-  const filteredElementFormState = Object.keys(elementFormState)
-    .filter(key => key.match(formNamePattern))
-    .reduce((accumulator, key) => ({
-      ...accumulator,
-      [key]: elementFormState[key].values
-    }), {});
-
-  return filteredElementFormState;
-});
-
-function mapStateToProps(state) {
-  // Memoize form state and value changes
-  const formState = elementFormSelector(state);
-
-  return { formState };
-}
```

Comment:
I don't want to have to parse all of this code to see what it's doing and try to infer why it was deleted - so I'll just ask. Why is this being deleted? What was it doing?

Reply:
The whole point of it was to extract the redux state and to it into a prop called `formState`

The only place that prop was used was to populate the now deleted `<input>` element whose purpose was to send elemental data in the POST request when clicking the page/parent save button